new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+    w      next word                   d      Delete text with motion
+    e      end of word                 y      Yank text with motion
+    b      beginning of word           c      Change text with motion
+    0      beginning of line           r      Replace character/text
+    $      end of line                 u      Undo action
+l <SPACE>  current character         <C-R>    Redo action
+    _      current line                p      Put yanked/deleted text
+    f      fing character           ========  ========================
+    t      'till character
+    i      inside something         Pressing an operator twice acts as
+    a      around something         a motion on the entire line.
+   gg      first line
+    G      last line
+    %      matching braces
+=========  =================
+
+
+
+# Part 2: EX COMMANDS
+
+** Command mode is where we can execute powerful commands **
+
+You have already met command-mode before when you quit Vim. First you press
+«:», then you enter your command and possibly some arguments and hit <CR> to
+execute it. Executing a command puts us back into normal mode.
+
+The name "ex command" has no particular reason, it is a historical leftover
+from a previous UNIX text editor called "ex". Ex was a predecessor to vi, which
+is a predecessor to Vim, and was operated exclusively using such commands.
+
+While in command mode you can auto-complete the same way as in the shell by
+hitting the <TAB> key. While typing a command you can press <C-D> to like
+possible suggestions.
+
+# Lesson 2.1: WRITING AND CLOSING A FILE
+
+** Use the ':write' command to write the current file and ':quit' to close it. **
+
+You have already met 'quit' when you quit Vim. If you have a file open you can
+use ':write' to save it to disc, assuming the file is not write-protected.
+
+To close a file use the ':quit' command. If the file has been edited Vim will
+not allow you to close it, this is to protect you from losing you changes by
+accidently closing. You can force a quit by supercharging the command with an
+exclamation mark: 'quit!'. This is a common convention, similar to the upper
+case for operators.
+
+Now it a good time to try it out. Close this file and then re-open it the same
+way you did last time.
+
+Typing out ':write' and ':quit' every time is cumbersome, so pretty much every
+ex command can be shortened. The shortes unambiguous part does the job, so you
+could also write ':wri' and get the same results. On top of that many commands
+also have special abbreviations. You will always see ':w' and ':q' written.
+
+To write and close a file in one go use ':wq'.
+
+You can close all open files with 'qall' or 'qa' and write all files with
+':wall' or ':wa'. And you might have guessed it already, use ':wqall' or 'wqa'
+to write and quit all files.
+
+Unlike actions there is not really a grammar to ex commands, but the commands
+have been chosen in a way that does look like a grammer where it makes sense.
```

Comment:
There are "micro grammars" for commands: ranges, regex arguments + flags, `|`... Perhaps remove this observation? I feel like it can be more confusing later on than anything.